app/views/incomeSources/cease/IncomeSourceCeasedObligations.scala.html

```diff
@@ -0,0 +1,212 @@
+@import java.time.LocalDate
+@import _root_.implicits.ImplicitDateFormatterImpl
+@import enums.IncomeSourceJourney.{ForeignProperty, IncomeSourceType, SelfEmployment, UkProperty}
+
+
+@this(
+  mainTemplate: unifiedLayout,
+  h1: h1,
+  implicitDateFormatter: ImplicitDateFormatterImpl,
+  continueButton: ContinueButton,
+  formWithCSRF: FormWithCSRF,
+  errorSummary: ErrorSummary
+)
+
+@(sources: ObligationsViewModel, backUrl: Option[String] = Some(""), isAgent: Boolean, incomeSourceType: IncomeSourceType, businessName: Option[String] = None)(implicit messages: Messages, user: auth.MtdItUser[_])
+
+@getMessage(key: String, args: String*) = @{
+  messages(s"business-ceased.obligation.$key", args: _*)
+}
+
+@businessNameFinal = @{ businessName.getOrElse(getMessage("heading1.se.part2")) }
+
+@getPageTitle = @{
+
+incomeSourceType match {
+        case  _:SelfEmployment.type =>
+            s"""${getMessage("heading1.se.part1")} ${businessNameFinal} ${getMessage("heading1.base")}""".trim()
+
+        case _:UkProperty.type =>
+            s"${getMessage("heading1.uk-property.part2")} " +
+            s"${getMessage("heading1.base")}".trim()
+
+        case _:ForeignProperty.type =>
+            s"""${getMessage("heading1.foreign-property.part2")} ${getMessage("heading1.base")}"""
+    }
+}
+
+@getCeaseIncomeSourceCall = @{
+    if(isAgent) {
+        controllers.incomeSources.cease.routes.CeaseIncomeSourceController.showAgent()
+    } else {
+        controllers.incomeSources.cease.routes.CeaseIncomeSourceController.show()
+    }
+}
+
+@bannerHtmlContent(head:String, h1:String, base:String) = {
+    <div class="govuk-panel__body govuk-!-margin-bottom-3">
+        @head
+    </div>
+    <h1 class="govuk-panel__title">
+        @h1
+    </h1>
+    <div class="govuk-panel__body">
+        @base
+    </div>
+}
+
+@getBannerContent() = @{
+    incomeSourceType match {
+        case _:SelfEmployment.type =>
+            bannerHtmlContent(getMessage("heading1.se.part1"),businessNameFinal,getMessage("heading1.base"))
+        case _:UkProperty.type =>
+            bannerHtmlContent("",getMessage("heading1.uk-property.part2"),getMessage("heading1.base"))
+        case _:ForeignProperty.type =>
+        bannerHtmlContent("",getMessage("heading1.foreign-property.part2"),getMessage("heading1.base"))
+    }
+}
+
+
+
+@import implicitDateFormatter.longDate
+@mainTemplate(pageTitle = getPageTitle,
+  backUrl = backUrl,
+  isAgent = isAgent,
+  btaNavPartial = user.btaNavPartial,
+  useFallbackBackLink = true) {
+
+    @formWithCSRF(action = getCeaseIncomeSourceCall) {
+
+        <div class="govuk-panel govuk-panel--confirmation" id="banner">
+            @getBannerContent()
+        </div>
+
+        <h2 class="govuk-label-wrapper">
+            <label class="govuk-label govuk-label--l govuk-!-margin-top-8">
+                @getMessage("heading2.text")
+            </label>
+        </h2>
+
+
+        @if(sources.quarterlyObligationsDatesYearOne.nonEmpty){
```

Comment:
Seems to be some repeated code in this view file from the other one, you could use a partial?